Minor edit to Plone Deployment sections (#696)
plone/training: docs/plone-deployment/new-project.md

For this training, we recommend you to provide:

- `project_title`: **Plone Conference**
- `github_organization`: Your GitHub username

> - `container_registry`: **2**

Reviewer: I am OK with this automatic editor change because it requires one fewer keystroke. :grin:

Author: I learned my lesson with autoformatting tools 🙃

```console
✨  Done in 98.97s.
```

> Remember, having the repository on GitHub is currently a requirement to build the Docker images automatically; this might be a good moment to do so in case you had not yet.

Reviewer: Provide a reference, and tweak wording.

This paragraph should be moved up almost to the top, after the page title at line 10, to become its introduction. I think is is good to first explain what we are going to do in this chapter, and why these choices were made.

```suggestion
As mentioned in the {doc}`intro`'s Training Choices, for this Training, GitHub is a requirement to build the Docker images automatically.
The steps in this training may be adapted to other providers, including GitLab.
```

> The same could be done with other options like GitLab, but we do not provide a working example for it at the moment.

Reviewer: We do not want to imply that the authors will write working examples in the trainings. If you have a working example that you would like to share, then please include it in this PR or another.

```suggestion
```

````{note}
Due to an output difference when translations are built by the `@plone/generator-volto`, at the moment, it is necessary to run on the root directory:

> ```{code-block} shell

Reviewer: `code-block` is necessary only when you want to use reStructuredText features, such as line numbering and emphasize lines.

````suggestion
```shell
````

make i18n
````

```

> ```

Reviewer: Please delete the empty triple backticks.

```console
✨  Done in 98.97s.
```

````{note}
Due to an output difference when translations are built by the `@plone/generator-volto`, at the moment, it is necessary to run on the root directory:

```shell

> make i18n

Reviewer: Close the triple backtick. Take care when nesting MyST blocks.
````suggestion
make i18n
```
````

Author: My intuition told me to close that backtick, but since the HTML version looked fine, I didn't. It's done now!

```console
✨  Done in 98.97s.
```

````{note}
Due to an output difference when translations are built by the `@plone/generator-volto`, at the moment, it is necessary to run on the root directory:

```shell
make i18n

> ````

Author: 🫣 Gee.. sorry about that one
